server.js

```diff
@@ -2942,7 +2942,11 @@ function objectWebServer() {
                                                 let deferred = false;
                                                 function finishFn(folderName) {
                                                     return async function() {
-                                                        await fsProm.rmdir(path.join(folderD, identityFolderName, 'target', folderName));
+                                                        try {
+                                                            await fsProm.rmdir(path.join(folderD, identityFolderName, 'target', folderName));
```

Comment:
Why is `{recursive: true}` needed sometimes and not others?

Reply:
Here it's because the folder is expected to be empty (with all of its contents processed by the renaming functions) and it would be evidence of an error condition if it weren't. This is definitely a special boy, most other uses of rmdir could get `recursive: true` and not explode